nomenclature/definitions/variable/emissions/emissions.yaml

```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emissions|CO2|Electricity:
+   description: CO2 emmissions of power plants .
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Biomass|Traditionnal:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Biomass|New and imported :
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Gas|OCGT:
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Gas|CCGT:
+   description: CO2 emmissions of a typical Combined Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Nuclear:
```

Comment:
what are the CO2 emissions of a nuclear power plant during normal operations?

Reply:
I understand that it is just written for consistency of dealing with all the conventional power plants

Reply:
yes this is a consistency variable.... emissions of nuclear ar 0.....In my model, as in most UC models and it may be the same in comillas, nuclear is modelled as a thermal plant then it has the same list of variables (see issue #59 that I created about the lists of technologies : there are groups of technologies that share the same list of variables); Not sure what is the best to do there

Reply:
should we propose the use of t for ton following the International System of Units?

Reply:
@arght yes I agree with t